docs/gulpfile.js

```diff
@@ -195,15 +195,15 @@ function createComponentExamples() {
 }
 
 const symlinks = parallel(
-  series(deleteComponentSymlinks, createComponentSymlinks),
-  series(deleteComponentImageSymlinks, createComponentImageSymlinks),
-  series(deleteUserManualSymlinks, createUserManualSymlinks)
+    series(deleteComponentSymlinks, createComponentSymlinks),
+    series(deleteComponentImageSymlinks, createComponentImageSymlinks),
+    series(deleteUserManualSymlinks, createUserManualSymlinks)
 );
 const nav = parallel(createComponentNav, createUserManualNav);
 const examples = series(deleteExamples, createUserManualExamples, createComponentExamples);
 
 exports.symlinks = symlinks;
 exports.nav = nav;
-exports.examples = examples;
-exports.default = series(symlinks, nav, examples);
-
+//exports.examples = examples;
```

Comment:
Perhaps we could add a comment here.
```suggestion
// We moved the examples to the `camel-examples` repository,
// this made sure that the examples were copied to the
// correct place for the documentation, according to the
// Antora directory structure. For now the examples are
// copied over from `camel-examples` manually.
//exports.examples = examples;
```